backend/tournesol/views/video.py

```diff
@@ -86,6 +88,10 @@ def get_queryset(self):
         request = self.request
         queryset = self.queryset
 
+        uploader = request.query_params.get('uploader')
+        if uploader:
+            queryset = queryset.filter(uploader=unquote(uploader))
```

Comment:
`request.query_params` is already responsible for decoding URL params, so `unquote` is unnecessary here and could event lead to incorrect values.